packages/server/modules/automate/migrations/20240523192300_add_is_test_automation_column.ts

```diff
@@ -0,0 +1,32 @@
+import { Knex } from 'knex'
+
+const AUTOMATIONS_TABLE = 'automations'
+const PLACEHOLDER_NULL_ID = 'null-execution-engine-automation-id'
```

Comment:
I'm not sure what the best practice is for `down` for "undo making a column nullable." There's a chance we've added `null` values since the `up` was run, right? This was the best I could find online for what to make those (but very open to anything else).

Reply:
you can also run a delete operation on those automation runs, the down migration is already a destructive mechanism, rendering the test automations broken, even if you run the up again.
In my time working on the server, we never had to run a down migration in production. So im not super worried about deleting some test automations

Reply:
roger, makes good sense to me